vendor/bat-native-ads/src/bat/ads/internal/client.cc

```diff
@@ -544,6 +544,17 @@ void Client::RemoveAllHistory() {
   SaveState();
 }
 
+std::string Client::GetVersionCode() const {
```

Comment:
Can we please change to `GetAppVersion`

Reply:
We specifically return `version_code`, why it should be called `GetAppVersion`?

Reply:
@samartnik because it is the app version, what if we added a schema version in the future?

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (10)